docs(api): exclude API endpoints for authorized-applications in KPI documentation DEV-845 (#6059)
kobotoolbox/kpi: kpi/docs/api/v2/authorized_applications/authenticate_user.md

## Authenticate a user by using the commandline

> * Note that there are two endpoint fo this documentation but that the twos are aliases

Reviewer: ```suggestion
* Note there are two endpoints for authenticating a user; they are aliases.
```
